docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Win32 App Isolation Introduction
+
+### Target Applications
+
+* Win32
+* Centennial
```

Comment:
Let's keep the naming consistent with https://learn.microsoft.com/en-us/windows/msix/desktop/desktop-to-uwp-behind-the-scenes#types-of-desktop-app so Desktop Bridge (Centennial).